storage/plugins/mobile-api/src/Controllers/CatalogController.php

```
@@ -365,6 +366,69 @@ public function genres(
+        try {
+            $rows = [];
+            // Static query, no user input; soft-delete honoured per CLAUDE.md rule 2.
+            $res = $this->db->query(
+                "SELECT MIN(TRIM(lingua)) AS language, COUNT(*) AS count
+                   FROM libri
+                  WHERE deleted_at IS NULL AND lingua IS NOT NULL AND TRIM(lingua) <> ''
+                  GROUP BY LOWER(TRIM(lingua))
+                  ORDER BY count DESC, language ASC"
+            );
+            if ($res !== false) {
+                while ($r = $res->fetch_assoc()) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)